Libraries/BNO/BNO.cpp

```diff
@@ -9,32 +9,14 @@
 
 BNO::BNO(Multiplexor *multiplexor) {
   I2C_ = multiplexor;
-  bno_ = Adafruit_BNO055();
+  // bno_ = Adafruit_BNO055();
```

Comment:
Delete line if not needed anymore.

Reply:
I also feel like you should print something if the bno was not initialized correctly, either to the lcd, or to serial print to let know the user that it didn't work. Or have another function like bool isInitialized() that lets return true if bno_.begin() was true, false otherwise.

So, in your constructor you'd have my_variable = bno_.begin();
And then, on isInitialized(): return my_variable and then you can handle that however you want.